src/display.rs

```diff
@@ -278,21 +278,16 @@ fn get_long_output(
 fn get_visible_width(input: &str) -> usize {
     let mut nb_invisible_char = 0;
 
-    for (idx, _) in input.match_indices("\u{1b}[38;5;" /* "\e[38;5;" */) {
-        let color_code = input.chars().skip(idx + 7);
-        let mut code_size = 0;
-        color_code
-            .skip_while(|x| {
-                code_size += 1;
-                char::is_numeric(*x)
-            })
-            .count();
-        nb_invisible_char += 6 + code_size; /* "\e[38;5;" + color number + "m" */
-    }
-
-    if nb_invisible_char > 0 {
-        // If no color have been set, the is no reset character.
-        nb_invisible_char += 3; /* "[0m" */
+    // Search for every instance of "\e[" indicating the start of a color code
+    // then count the number of charactere between the string and the first 'm'
+    // charactere indicating the end of the code color.
+    for (idx, _) in input.match_indices("\u{1b}[") {
+        for c in input.chars().skip(idx) {
```

Comment:
I feel like using skip will have issues with non unicode ones.
Unicode length might be more than just simple length.